eregs_core/management/commands/cache_content.py

```diff
@@ -0,0 +1,59 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
Can you please order imports as suggested by [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), i.e. putting standard library imports before local ones?